nbs/01_features.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # |default_exp features
</pre><p>This should be <code>#| default_exp tsfeatures</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/tsfeatures/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>